### DISCUSSION TITLE
Now-a-days the codespace is unable to forward port or make any port.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#45353] [Codespaces]
### Select Topic Area

Product Feedback

### Body

I am unable to make the webserver. Sometime is works but most of the time it doesn't.
![image](https://user-images.githubusercontent.com/108215141/214616396-eaa69a6b-7503-4784-a4d9-3b84b7243699.png)

**Comment by dmilburn** [comment]
Hi @Swayam25, i'm sorry for the inconvenience and thanks for reaching out. A few questions:

- Could you share the name of the codespace or repo where this is happening? 
- Does anyone else work on this project with you? Has anyone else had the same experience?
- Do you have any dotfiles enabled?
- From the title of your post, it sounds like this has been recent. Do you have a sense about when this started?
- How do you connect to codespaces? In the web? Through VSCode? CLI?

**Reply by ghost** [reply]
1. Repo is private.
2. No, I am working alone.
3. No dotfiles.
4. It started from 1st January 2023 (according to my notice).
5. I use VSCode Web.
